app/controllers/reports_controller.rb

```diff
@@ -2,31 +2,23 @@ class ReportsController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
 
   before_filter :setup_search_options, :only => :index
+  before_filter :deprecated
```

Comment:
why do we need to deprecate a ui controller? can't we fix it in apache/routes instead of keeping a controller to redirect?